msdsn/conn_str.go

```diff
@@ -183,6 +195,28 @@ func Parse(dsn string) (Config, map[string]string, error) {
 		}
 	}
 
+	krb5ConfFile, ok := params["krb5conffile"]
```

Comment:
you will move all this code into a separate kerb auth provider, correct? The idea is to allow apps to include that package if desired rather than having it in the core.